apps/src/templates/sectionProgress/SectionProgress.jsx

```diff
@@ -246,23 +196,20 @@ class SectionProgress extends Component {
               className="fa-pulse fa-3x"
             />
           )}
-          {levelDataInitialized && (
+          {levelDataInitialized && currentView === ViewType.SUMMARY && (
```

Comment:
the astute observer will note that we previously were rendering all three tables at once, regardless of which was visible. we get a nice extra performance boost just with this simple change!